code/BuddyTrackerBasic/src/main.cpp

```diff
@@ -180,12 +186,12 @@ void handleGPS(){
 
 
 // returns 0 if no match
-uint8_t findBuddyIndex(uint64_t UUID){
+uint8_t findBuddyBy(uint64_t UUID){
```

Comment:
findBuddy returns an index, it does not find by index. Can change name to just findBuddyBy since UUID, the key, is the first param name. Or change to findBuddyByUUID.